scripts/get_new_db_name.pl

```diff
@@ -28,9 +28,13 @@
     my $v1 = $2 + 1;
     my $v2 = $3 + 1;
     print "${1}${v1}_${v2}${4}\n";
+} elsif ($name =~  m/^(.*_compara_[a-z]+_)([0-9]+)_([0-9]+)$/) {
+    my $v1 = $2 + 1;
+    my $v2 = $3 + 1;
+    print "${1}${v1}_${v2}\n";
 } elsif($name =~ m/^(.*_[a-z]+_)([0-9]+)_([0-9]+)$/) {
     my $v1 = $2 + 1;
     print "${1}${v1}_${3}\n";
 } else {
-    print "$name\n";
+    print "4 $name\n";
```

Comment:
```suggestion
    print "$name\n";
```
Fix to report to main